app/services/foreman_templates/template_importer.rb

```diff
@@ -54,54 +51,52 @@ def parse_files!
         result_lines << 'Parsing: ' + template.gsub(/#{@dir}#{@dirname}/, '') if @verbose
 
         metadata = parse_metadata(text)
```

Comment:
Shouldn't we rather use `Template::parse_metadata`?

Reply:
:1st_place_medal: